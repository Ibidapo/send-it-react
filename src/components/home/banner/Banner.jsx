import React from 'react';
import { Link } from 'react-router-dom';

import './styles.scss';

const Banner = () => (
  <div id="banner">
    <div className="intro w-60 mx-auto text-center">
      <h1 className="white p-1 mt-0">SendIT is the #1 courier service that helps users deliver parcels to different
        destinations all across the country.</h1>
      <Link to='#content' className="purple-btn">Get Started</Link>
    </div>
  </div>
);

export default Banner;