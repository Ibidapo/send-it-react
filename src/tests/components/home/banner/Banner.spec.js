import React from 'react';
import { shallow } from 'enzyme';

import Banner from '../../../../components/home/banner/Banner';

describe('Test for Footer Component', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<Banner />);
    expect(wrapper).toMatchSnapshot();
  })
});