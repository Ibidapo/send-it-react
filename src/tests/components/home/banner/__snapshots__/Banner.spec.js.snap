// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Footer Component Should render correctly 1`] = `
<div
  id="banner"
>
  <div
    className="intro w-60 mx-auto text-center"
  >
    <h1
      className="white p-1 mt-0"
    >
      SendIT is the #1 courier service that helps users deliver parcels to different destinations all across the country.
    </h1>
    <Link
      className="purple-btn"
      to="#content"
    >
      Get Started
    </Link>
  </div>
</div>
`;
